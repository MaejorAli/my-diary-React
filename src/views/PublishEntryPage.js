import React from 'react';
import PublishEntry from '../containers/PublishEntry';


export default (props) => (
  <div>
    <PublishEntry {...props} />
  </div>
);
