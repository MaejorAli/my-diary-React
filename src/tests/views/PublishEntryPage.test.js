import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PublishEntryPage from '../../views/PublishEntryPage';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<PublishEntryPage />);
});


describe('Publish Entry Page UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
