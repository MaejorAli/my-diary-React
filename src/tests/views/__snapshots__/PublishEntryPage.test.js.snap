// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Publish Entry Page UI render features view should render as expected 1`] = `
<div>
  <Connect(PublishEntry) />
</div>
`;
